In-house patch to remove unnecessary dependencies from Heat's
requirements files. The specific reasons are as follows:

pycrypto	Not applicable to Solaris (M2Crypto used instead)

--- heat-6.0.0/requirements.txt.~1~	2016-04-07 00:37:15.000000000 -0700
+++ heat-6.0.0/requirements.txt	2016-06-10 14:59:42.585163120 -0700
@@ -29,7 +29,6 @@ oslo.utils>=3.5.0 # Apache-2.0
 osprofiler>=1.1.0 # Apache-2.0
 oslo.versionedobjects>=1.5.0 # Apache-2.0
 PasteDeploy>=1.5.0 # MIT
-pycrypto>=2.6 # Public Domain
 python-barbicanclient>=3.3.0 # Apache-2.0
 python-ceilometerclient>=2.2.1 # Apache-2.0
 python-cinderclient>=1.3.1 # Apache-2.0
